src/Tools/dotnet-monitor/ServiceCollectionExtensions.cs

```diff
@@ -240,6 +241,25 @@ public static IServiceCollection ConfigureStorage(this IServiceCollection servic
             return services;
         }
 
+        public static IServiceCollection ConfigureCapabilities(this IServiceCollection services, bool noHttpEgress)
+        {
+            services.AddSingleton(new MonitorCapability() { Name = MonitorCapabilityConstants.Metrics });
```

Comment:
nit: Consider creating a ConfigureCapability function that handles the necessary singleton registration for a single capability that requires post configuration.